packages/config/src/bridges/lzOmnichain-config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "comment": "This file was autogenerated. Do not modify it by hand, instead use: yarn update-omnichain",
+  "escrows": [
+    {
+      "address": "0xAf5191B0De278C7286d6C7CC6ab6BB8A73bA2Cd6",
```

Comment:
Shouldn't this be part of https://l2beat.com/bridges/projects/stargate?

Reply:
I am not sure about this one, maybe @luke7211 would know the answer